uspv/msghandler.go

```diff
@@ -13,7 +13,8 @@ func (s *SPVCon) incomingMessageHandler() {
 		n, xm, _, err := wire.ReadMessageWithEncodingN(s.con, s.localVersion,
 			wire.BitcoinNet(s.Param.NetMagicBytes), wire.LatestEncoding)
 		if err != nil {
-			log.Printf("ReadMessageWithEncodingN error.  Disconnecting: %s\n", err.Error())
+			log.Printf("ReadMessageWithEncodingN error.  Disconnecting from given peer. %s\n", err.Error())
+			s.Connect("yes") // really any YupString here
```

Comment:
I think we need a check here.  If someone starts with `--tn3 127.0.0.1` they really only want it to connect to localhost.  This would mean that if the connection to localhost failed (after getting established) their node would attempt to connect to random nodes, without the user knowing.

If the user supplies a fullnode, and that node doesn't work, the program's response should be to fail & inform the user, not try other nodes.

Reply:
yeah, I was wondering what to do about it too, I guess I'll  whitelist `127.0.0.1` and `localhost` for now (until a better mechanism)